app/javascript/controllers/webauthn_credential_controller.js

```diff
@@ -0,0 +1,60 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["hiddenCredentialInput"]
+  static values = { optionsUrl: String }
+
+  async create() {
+    try {
+      const optionsResponse = await fetch(this.optionsUrlValue, {
+        method: "POST",
+        body: new FormData(this.element),
+        headers: {
+          "X-CSRF-Token": document.querySelector('meta[name="csrf-token"]')?.getAttribute("content")
```

Comment:
Perhaps we could use [`requestjs-rails`](https://github.com/rails/requestjs-rails) so that we don't have to care about this? What do you think?